lib/manager/terraform/lockfile/index.ts

```diff
@@ -0,0 +1,106 @@
+import pMap from 'p-map';
+import { GetPkgReleasesConfig, getPkgReleases } from '../../../datasource';
+import { logger } from '../../../logger';
+import { get as getVersioning } from '../../../versioning';
+import type {
+  UpdateArtifact,
+  UpdateArtifactsConfig,
+  UpdateArtifactsResult,
+} from '../../types';
+import hash from './hash';
+import type { ProviderLock, ProviderLockUpdate } from './types';
+import {
+  extractLocks,
+  isPinnedVersion,
+  readLockFile,
+  writeLockUpdates,
+} from './util';
+
+async function updateAllLocks(
+  locks: ProviderLock[],
+  config: UpdateArtifactsConfig
+): Promise<ProviderLockUpdate[]> {
+  const updates = await pMap(
+    locks,
+    async (lock) => {
+      const updateConfig: GetPkgReleasesConfig = {
+        versioning: 'hashicorp',
+        datasource: 'terraform-provider',
+        depName: lock.lookupName,
+      };
+      const releasesResult = await getPkgReleases(updateConfig);
+      const releases = releasesResult.releases;
```

Comment:
```suggestion
      const { releases } = await getPkgReleases(updateConfig);
```

just a small simplification